docs/plugins/index.md

```diff
@@ -26,7 +26,7 @@ Check out [Using Plugins](../guide/using-plugins) for information on how to use
 
```

Comment:
If we are making it clear that SWC won't be used in those cases, I think we should update the plugin-react section, too. IIRC plugin-react doesn't use babel during build when no plugin is configured.

Reply:
Yes, a good opportunity to clarify that. I don't think it is clear to many users.